Force building with *spring* Creators Update SDK

The Fall Creators Update SDK has various bugs/incompatibilities that
cause errors when building Chromium. This will not affect Google
employees who use a packaged toolchain but good affect anybody who set
DEPOT_TOOLS_WIN_TOOLCHAIN=0 and is therefore using the system install
of VS 2017. This change forces vcvarsall.bat to use the 15063 SDK
instead of the latest-installed.

This should be a temporary measure that can be undone as we land
workarounds for the incompatibilities or get fixed SDK versions.

diff --git a/build/toolchain/win/setup_toolchain.py b/build/toolchain/win/setup_toolchain.py
index 8150d3a4e4ac..bb599d62968f 100644
--- a/build/toolchain/win/setup_toolchain.py
+++ b/build/toolchain/win/setup_toolchain.py
@@ -139,9 +139,11 @@ def _LoadToolchainEnv(cpu, sdk_dir):
         raise Exception('%s is missing - make sure VC++ tools are installed.' %
                         script_path)
       script_path = other_path
-    # Chromium requires the 10.0.14393.0 SDK or higher - previous versions don't
-    # have all of the required declarations.
-    args = [script_path, 'amd64_x86' if cpu == 'x86' else 'amd64']
+    # Chromium requires the 10.0.15063.468 SDK - previous versions don't have
+    # all of the required declarations and 10.0.16299.0 has some
+    # incompatibilities (crbug.com/773476).
+    args = [script_path, 'amd64_x86' if cpu == 'x86' else 'amd64',
+            '10.0.15063.0']
     variables = _LoadEnvFromBat(args)
   return _ExtractImportantEnvironment(variables)
 
diff --git a/build/vs_toolchain.py b/build/vs_toolchain.py
index 3b2c727f51d5..9b005fcc5d57 100755
--- a/build/vs_toolchain.py
+++ b/build/vs_toolchain.py
@@ -346,8 +346,9 @@ def _GetDesiredVsToolchainHashes():
     # Update 3 final with 10.0.15063.468 SDK and no vctip.exe.
     return ['f53e4598951162bad6330f7a167486c7ae5db1e5']
   if env_version == '2017':
-    # VS 2017 Update 3 Preview 4 with 10.0.15063.468 SDK.
-    return ['1f52d730755ac72dddaf121b73c9d6fd5c24ddf8']
+    # VS 2017 Update 3.2 with 10.0.15063.468 SDK, patched setenv.cmd, and
+    # 10.0.16299.15 debuggers.
+    return ['1180cb75833ea365097e279efb2d5d7a42dee4b0']
   raise Exception('Unsupported VS version %s' % env_version)
 
 
